vizro-core/docs/pages/examples/your-examples.md

```diff
@@ -22,11 +22,14 @@ If you have something you'd like us to include on the list, or spot something th
 ## Blog posts
 * [Introducing Vizro](https://quantumblack.medium.com/introducing-vizro-a-toolkit-for-creating-modular-data-visualization-applications-3a42f2bec4db).
 * [Creating Custom Dashboards with Vizro: A Comprehensive Guide](https://medium.com/@saffand03/creating-custom-dashboards-with-vizro-a-comprehensive-guide-73c69c6f851e).
+* [I built a reusable dashboard read-the-docs traffic analytics using vizro](https://medium.com/towards-data-science/i-built-a-reusable-dashboard-for-read-the-docs-traffic-analytics-using-vizro-47dc15dc04f8).
```

Comment:
@stichbury - I think these were the only new blog posts right?

Reply:
There's one more -- I've added it.